html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <form action="" method="get" class="form">
+      <div class="form">
+        <label for="email">Email</label>
+        <br />
+        <input id="email" type="text">
+        </div>
+      <div>
+        <label for="why is coding cool">Why is coding cool?</label>
+        <br />
+        <select name="reasons">
+          <option value="select an option">Select and option</option>
+          <option value="because it's interesting">Because it's interesting</option>
+          <option value="because it's fun">Because it's fun</option>
+          <option value="because it's powerful">Because it's powerful</option>
+        </select>
+        </div>
+        <div>
+          <label for="favorite language">Favorite Language (choose one)</label>
+          <br />
+          <input type="radio" name="language" value="html" />HTML
+          <input type="radio" name="language" value="css" />CSS
+          <input type="radio" name="language" value="javascript" />JavaScript
+        </div>
+        <div>
+          <label>Would you like to sign up for more information?</label>
+          <br />
+          <input type="checkbox" name="sign me up" value="sign me up">Sign me up!
+        </div>
+        <div>
+          <label for="comments">Comments</label>
```

Comment:
Your `for` attribute is correct here, but it needs to correlate to an `id` on the target input.